azure-pipelines.yml

```diff
@@ -37,7 +39,7 @@ stages:
   jobs:
   - job: SynapseE2ETest
     steps:
-    - template: FhirToDataLake/synapse-e2e-test.yml
+    - template: FhirToDataLake/azurepipelines/synapse-e2e-test.yml
 
   - job: PublishFhirToCdm
```

Comment:
just curious: why `PublishFhirToCdm` dependsOn `SynapseE2ETest`?

Reply:
just want to make sure the E2E tests passed before releasing~